.github/workflows/ci-cd.yml

```
@@ -50,16 +50,72 @@ jobs:
+            cp next.config.js artifact-bundle/
+          else
+            echo "ERROR: next.config.js not found"
+            exit 1
+          fi
+          
+          echo "All required paths exist, proceeding with artifact upload."
+      - name: List build output
+        run: ls -la .next/
+
       - name: Archive production artifacts
         uses: actions/upload-artifact@v3
```

actionlint (1.7.4)

the runner of "actions/upload-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)